env/env_posix.cc

```diff
@@ -428,6 +420,15 @@ PosixEnv::PosixEnv()
   thread_status_updater_ = CreateThreadStatusUpdater();
 }
 
+PosixEnv::PosixEnv(const PosixEnv* default_env, std::shared_ptr<FileSystem> fs)
```

Comment:
I do not think this constructor is or should be necessary.  Who calls it and when?